R/MSnExperiment-class.R

```diff
@@ -99,7 +129,44 @@ NULL
 #'
 #' @examples
 #'
-#' ## TODO
+#' ## Create an MSnExperiment from two input files using the on-disk
+#' ## BackendMzR backend
+#' sf <- dir(system.file("sciex", package = "msdata"), full.names = TRUE)
+#' dta <- readMSnExperiment(sf, backend = BackendMzR())
+#' dta
+#'
+#' ## Get associated file names
+#' fileNames(dta)
+#'
+#' ## Get spectra metadata
+#' spectraData(dta)
+#'
+#' ## Extract all spectra; by default a `Spectra` is returned. We could also
+#' ## get a simple `list` of `Spectrum` objects by specifying
+#' ## `return.type = "list"`.
+#' spctra <- spectra(dta)
```

Comment:
`spctra` seems to be a wired variable name in an example, IMHO even `s` or `sp` would be better (or remove all vowels: `spctr` :wink:)

Reply:
Fixed.